src/smk/viewer.js

```diff
@@ -464,11 +464,11 @@ include.module( 'viewer', [ 'jquery', 'util', 'event', 'layer', 'feature-set', '
                 } )
                 .then( function ( features ) {
                     features.forEach( function ( f, i ) {
-                        if ( ly.config.titleAttribute ) {
-                            var m = ly.config.titleAttribute.match( /^(.+?)(:[/](.+)[/])?$/ )
+                        if ( ly.config.attributes[0].id ) {
```

Comment:
This change is not needed. The layer config titleAttribute is used to determine what feature attr is used as a the title for the feature. This should be fixed in the layer config.

Reply:
https://vividsolutions.atlassian.net/secure/RapidBoard.jspa?rapidView=145&modal=detail&selectedIssue=WFONE-367&assignee=5d5ecf7b7b9a8f0cf55fd2a4

That is the issue that i am trying to resolve. It looks like the 'title' is being used to fill up that field. Is there another place that handles that portion? I don't think I saw it.... is it jus the layer.js code?